docs/products/postgresql/reference/list-of-extensions.md

```diff
@@ -1,26 +1,31 @@
 ---
 title: Extensions on Aiven for PostgreSQL®
+sidebar_label: Extensions
 ---
 
-The following PostgreSQL® extensions are available. Please note that
-some of the extensions have dependencies and they need to be created in
-the proper order. Some extensions may require disconnecting the client
-connection and reconnecting before they are fully available.
+PostgreSQL® extensions allow you to extend the functionality by adding more capabilities to your Aiven for PostgreSQL.
 
-To check the details, including the version number of the extension, run
-`select * from pg_available_extensions` in your Aiven for PostgreSQL
-server.
+:::important
+
+- Some extensions have dependencies and need to be created in a predetermined order.
+- There are extensions that require resetting the client connection before they are fully
+  available.
```

Comment:
Why not harmonize "some extensions" and "there are extensions" since it's the same thing and then you can "factorize" it:

```suggestion
Some extensions:
- Have dependencies and need to be created in a predetermined order.
- Require resetting the client connection before they are fully
  available.
```